File: 21451077闭鸿晖/HomeWork_TaskList/HomeWork_TaskList/EditViewController.m
//
//  EditViewController.m
//  HomeWork_TaskList
//
//  Created by turbobhh on 11/9/14.
//  Copyright (c) 2014 org.bhh.homework. All rights reserved.
//

#import "EditViewController.h"

@interface EditViewController ()
@property (weak, nonatomic) IBOutlet UITextField *taskTextField;

@end

@implementation EditViewController

-(void)viewWillAppear:(BOOL)animated{
    self.taskTextField.text=self.preTaskLabel.text;
}

- (void)viewDidLoad {
    [super viewDidLoad];
    self.navigationController.navigationBarHidden=NO;
    self.navigationItem.title=@"编辑任务名";

}

- (void)didReceiveMemoryWarning {
    [super didReceiveMemoryWarning];
    // Dispose of any resources that can be recreated.
}

/*
#pragma mark - Navigation

// In a storyboard-based application, you will often want to do a little preparation before navigation
- (void)prepareForSegue:(UIStoryboardSegue *)segue sender:(id)sender {
    // Get the new view controller using [segue destinationViewController].
    // Pass the selected object to the new view controller.
}
*/
- (IBAction)confirm:(id)sender {
    self.preTaskLabel.text=self.taskTextField.text;
     [self.navigationController popToRootViewControllerAnimated:YES];
}

- (IBAction)cancel:(id)sender {
    [self.navigationController popToRootViewControllerAnimated:YES];
}


@end

File: 21451122陈斌/MyNotesOne/MyNotesOne/EditViewController.swift
//
//  EditViewController.swift
//  MyNotesOne
//
//  Created by lqynydyxf on 14/11/26.
//  Copyright (c) 2014年 lqynydyxf. All rights reserved.
//

import UIKit
import CoreData
class EditViewController: UIViewController {

    @IBOutlet weak var editTitle: UITextField!
    
    @IBOutlet weak var editContent: UITextView!
    
    var data:NSManagedObject!
    
    override func viewDidLoad() {
        super.viewDidLoad()
        editContent.layer.borderColor = UIColor.grayColor().CGColor
        editContent.layer.borderWidth = 1.0
        editContent.layer.cornerRadius = 5.0
        editTitle.text = data.valueForKey("title") as String
        editContent.text = data.valueForKey("content") as String
    }

    override func didReceiveMemoryWarning() {
        super.didReceiveMemoryWarning()
        // Dispose of any resources that can be recreated.
    }
    
    @IBAction func save(sender: AnyObject) {
        
        data.setValue(editTitle.text, forKey: "title")
        data.setValue(editContent.text, forKey: "content")
        data.setValue(NSDate(), forKey: "time")
        data.managedObjectContext?.save(nil)
        dismissViewControllerAnimated(true, completion: nil)
    }

    @IBAction func cancel(sender: AnyObject) {
        dismissViewControllerAnimated(true, completion: nil)
    }
    /*
    // MARK: - Navigation

    // In a storyboard-based application, you will often want to do a little preparation before navigation
    override func prepareForSegue(segue: UIStoryboardSegue, sender: AnyObject?) {
        // Get the new view controller using segue.destinationViewController.
        // Pass the selected object to the new view controller.
    }
    */

}
